File: app/src/main/java/ru/coffeeplanter/androidlogin/presentation/fragments/loggedin/LoggedInView.java
package ru.coffeeplanter.androidlogin.presentation.fragments.loggedin;

import ru.coffeeplanter.androidlogin.presentation.BaseView;

interface LoggedInView extends BaseView {

    void getDataFromFragmentArguments();
    void setGreetingMessage();
    void closeApp();
    void navigateToLoginFragment();

}

File: app/src/main/java/ru/coffeeplanter/androidlogin/presentation/fragments/loggedin/LoggedInView.kt
package ru.coffeeplanter.androidlogin.presentation.fragments.loggedin

import ru.coffeeplanter.androidlogin.presentation.BaseView

internal interface LoggedInView : BaseView {

    fun getDataFromFragmentArguments()
    fun setGreetingMessage()
    fun closeApp()
    fun navigateToLoginFragment()

}
